packages/visualizations/addon/helpers/format-table-cell.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import { helper } from '@ember/component/helper';
+import { isEmpty } from '@ember/utils';
+import numeral from 'numeral';
+import { smartFormatNumber } from 'navi-core/helpers/smart-format-number';
+
+export function formatTableCell([value, format, smartFormat]) {
```

Comment:
Can we call the boolean argument `shouldSmartFormat`?
And can we add jsdoc for this function?